synthetic_phi/whole_slide_image/src/isyntax.py

```diff
@@ -0,0 +1,159 @@
+import os
+import traceback
+
+from lxml import etree as ET
+import base64
+import io
+from PIL import Image, ImageDraw
+
+def generate_image(x, y, image_text1=None, image_text2=None):
+    img = Image.new('RGB', (x, y), color=(73, 109, 137))
+
+    d = ImageDraw.Draw(img)
+    d.text((100, 100), "REMOVED DATA 1", fill=(255, 255, 0))
+    d.text((100, 130), "REMOVED DATA 2", fill=(255, 255, 0))
+
+    with io.BytesIO() as output:
+        img.save(output, format='JPEG')
+        contents = output.getvalue()
+        base64_bytes = base64.b64encode(contents)
+        base64_string = base64_bytes.decode("utf-8")
+
+    return base64_string
+
+def base64Split(a_string):
+
+    split_strings = []
+    n = 60
+    for index in range(0, len(a_string), n):
+        split_strings.append(a_string[index: index + n])
+
+    payload = ''
+    for str in split_strings:
+        payload += str + '\n'
+
+    return payload[:-1]
+
+def parseXMLheader(header_data, image_text1=None, image_text2=None):
```

Comment:
Are you following coding style where local functions are using camel case? if not, can you rename above function name as parse_xml_header?

Reply:
Also, what is image_text1 and 2?